src/MetaMorpho.sol

```diff
@@ -296,13 +296,15 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         emit EventsLib.SetSupplyQueue(_msgSender(), newSupplyQueue);
     }
 
-    /// @notice Sets the withdraw queue as a permutation of the previous one, although markets with zero cap and zero
-    /// vault's supply can be removed.
+    /// @notice Sets `WithdrawQueue` as a new withdraw queue defined by `indexes`.
+    /// @notice This is the only entry point to disable a market.
+    /// @notice Sets the withdraw queue as a permutation of the previous one, although markets with both zero cap and
+    /// zero vault's supply can be removed from the permutation to be disabled.
     /// @notice Removing a market requires the vault to have 0 supply on it; but anyone can supply on behalf of the
     /// vault so the call to `sortWithdrawQueue` can be griefed by a frontrun. To circumvent this, the allocator can
     /// simply bundle a reallocation that withdraws max from this market with a call to `sortWithdrawQueue`.
     /// @param indexes The indexes of each market in the previous withdraw queue, in the new withdraw queue's order.
-    function sortWithdrawQueue(uint256[] calldata indexes) external onlyAllocatorRole {
+    function setWithdrawQueue(uint256[] calldata indexes) external onlyAllocatorRole {
```

Comment:
I understand why `sort` is not best-suited
I also think that `set` is less suited

What should we do?

Reply:
As markets can be removed, this function doesn't always just "sort" the queue, the the issue requested to change the naming.

Reply:
Yes I understand but still, I think that `set` is less suited and even misleading compared to `sort`

Reply:
Great!